spec/spec_helper.rb

```diff
@@ -120,7 +120,8 @@
 
   c.allow_http_connections_when_no_cassette = false
   c.default_cassette_options = {
-    :allow_unused_http_interactions => false
+    :allow_unused_http_interactions => false,
+    :decode_compressed_response     => true
```

Comment:
Without this change content of recorded http calls is gzipped. I found this useful to have because I wanted to see content in plain text. If you think it is not needed I can remove it.